apps/crawler/src/crawler.service.ts

```diff
@@ -0,0 +1,95 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { DataSource as SubscquidDataSource } from '@subsquid/substrate-processor';
+import { Range } from '@subsquid/substrate-processor/lib/util/range';
+import { DataSource } from 'typeorm';
+import { CollectionsProcessor } from './processors/collections-processor';
+import { TokensProcessor } from './processors/tokens-processor';
+
+@Injectable()
+export class CrawlerService {
+  constructor(
+    private logger: Logger,
+    private dataSource: DataSource,
+    private collectionsProcessor: CollectionsProcessor,
+    private tokensProcessor: TokensProcessor,
+  ) {
+    this.logger = new Logger('CrawlerService');
```

Comment:
Another way without string literals. https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging

Reply:
I knew there is a more feng shui way! Thanks!